olcut-core-test/src/test/java/com/oracle/labs/mlrg/olcut/test/config_tests/StringMultiwordTest.java

```diff
@@ -0,0 +1,24 @@
+package com.oracle.labs.mlrg.olcut.test.config_tests;
```

Comment:
Missing copyright & license header.

Reply:
Added license.  Burned by the IDE hiding the license all the time!